modules/GoogleApi.js

```diff
@@ -62,6 +72,8 @@ class GoogleApi extends React.Component {
         clientId: this.props.clientId,
         discoveryDocs: this.props.discoveryDocs,
         scope: this.props.scopes.join(' '),
+        fetch_basic_profile: this.props.profile,
+        redirect_uri: 'http://localhost:3000'
```

Comment:
If `this.props.profile` isn't set, what happens? `redirect_uri` should also be configurable by props